task `generate:struct2interface` is broken because of `examples/infra/redis/kod_gen_interface.go`

```console
$ task generate:struct2interface
task: Task "install:mockgen" is up to date
task: [generate:struct2interface] go run ./cmd/kod struct2interface .
2024/06/08 08:30:42 Loading input failed: examples/infra/redis/kod_gen_interface.go:11:8: failed parsing returns: examples/infra/redis/kod_gen_interface.go:11:12: unknown package "redis"
panic: mockgen error: exit status 1

goroutine 1 [running]:
github.com/samber/lo.must({0x7b1d80, 0xc000417610}, {0x0, 0x0, 0x0})
	/home/whatever/go/pkg/mod/github.com/samber/lo@v1.39.0/errors.go:53 +0x1df
github.com/samber/lo.Must0(...)
	/home/whatever/go/pkg/mod/github.com/samber/lo@v1.39.0/errors.go:72
github.com/go-kod/kod/cmd/kod/internal.init.func4(0xb8a660, {0xc000032420?, 0x4?, 0x81a17c?})
	/home/whatever/Documents/dev/perso/kod/cmd/kod/internal/struct2interface.go:379 +0xa9
github.com/spf13/cobra.(*Command).execute(0xb8a660, {0xc0000323e0, 0x1, 0x1})
	/home/whatever/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xab1
github.com/spf13/cobra.(*Command).ExecuteC(0xb8a380)
	/home/whatever/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/whatever/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/go-kod/kod/cmd/kod/internal.Execute()
	/home/whatever/Documents/dev/perso/kod/cmd/kod/internal/root.go:33 +0x1a
main.main()
	/home/whatever/Documents/dev/perso/kod/cmd/kod/main.go:6 +0xf
exit status 2
task: Failed to run task "generate:struct2interface": exit status 1
```

infra/redis package imports "github.com/redis/go-redis/v9"

but Go reports the file as missing, calling `task mod` doesn't solve issue

--- a/examples/infra/redis/kod_gen_interface.go
+++ b/examples/infra/redis/kod_gen_interface.go
@@ -3,10 +3,10 @@
 package redis
 
 import (
-	"github.com/redis/go-redis/v9"
+	goredis "github.com/redis/go-redis/v9"
 )
 
 // impl is a component that implements Component.
 type Component interface {
-	Client() *redis.Client
+	Client() *goredis.Client
 }

--- a/examples/infra/redis/redis.go
+++ b/examples/infra/redis/redis.go
@@ -5,14 +5,14 @@ import (
 
 	"github.com/go-kod/kod"
 	"github.com/go-kod/kod/ext/client/kredis"
-	"github.com/redis/go-redis/v9"
+	goredis "github.com/redis/go-redis/v9"
 )
 
 type impl struct {
 	kod.Implements[Component]
 	kod.WithConfig[config]
 
-	cc *redis.Client
+	cc *goredis.Client
 }
 
 type config struct {
@@ -24,6 +24,6 @@ func (ins *impl) Init(ctx context.Context) error {
 	return nil
 }
 
-func (ins *impl) Client() *redis.Client {
+func (ins *impl) Client() *goredis.Client {
 	return ins.cc
 }

